### PR TITLE
Improve CLI usage docs

### PR DESCRIPTION
**Pull request by Andersson007**
usage.rst: use version 3, use single-file approach

### CODE REVIEW

**Review by cidrblock — APPROVED** [review]
awesome.  I think the single file approach is the smoothest path for most

**Comment by Andersson007** [comment]
@cidrblock someone changed version to 3 in parallel.
the conflicts are solved, could you please merge if it's OK now

**Review by Shrews — COMMENTED** [review]
Thanks for the PR. I missed that this was changed from version 1 to version 3 in the recent docs update.

I'm sort of conflicted on this dated example even being here because:
  - It repeats information that we've more completely defined in the EE def section.
  - It encourages using the default base image (which is there for backward compatibility for older EE versions) which is no longer maintained and will likely cause issues for folks who may choose to expand on this example.

I'd actually be in favor of removing this example and just linking the reader to the V3 docs in `docs/definition.rst`. Otherwise, if we want to keep this example, it should probably be expanded to include a different base image and any other options required for it to build properly.

**Comment by Andersson007** [comment]
@Shrews hello, thanks for reviewing!

- removed the example
- restructured the paragraph a bit
- fixed a typo

PTAL

File: docs/usage.rst [hunk 1]
```diff
@@ -11,34 +11,22 @@ Ansible Builder can execute two separate steps. The first step is to create a bu
...
 
-To build an execution environment using the files above, run:
+To build an execution environment using the default definition file, run:
 
 .. code::
 
    $ ansible-builder build
    ...
-   STEP 7: COMMIT my-awx-ee
+   STEP 7: COMMIT my-ee
```
Comment (Shrews) [inline docs/usage.rst]: Technically, this output isn't correct, especially the container tag name, which is not the default. Unless the `-v` verbosity option is used, the only thing the user will see with this command is (with slight modification to context path):

```
Running command:
  podman build -f context/Containerfile -t ansible-execution-env:latest context
Complete! The build context can be found at: /path/to/context
```

Comment (Andersson007) [reply docs/usage.rst]: so can i just update the output showing this ^ ?

Comment (Shrews) [reply docs/usage.rst]: > so can i just update the output showing this ^ ?

Sure

Comment (Andersson007) [reply docs/usage.rst]: thanks, done, PTAL

**Review by Shrews — APPROVED** [review]

**Comment by Andersson007** [comment]
@cidrblock @Shrews thank you!

**Pull request merged**